docs/as-a-resource/mapping-property-names.md

```diff
@@ -80,3 +80,36 @@ And a transformed version of the data object will look like this:
     'record_company' => 'RCA Records',
 ]
 ```
+
+### Available mappers:
+
+```php
+class ContractData extends Data
+{
+    public function __construct(
+        #[MapName(CamelCaseMapper::class)]
+        public string $name,
+        #[MapName(SnakeCaseMapper::class)]
+        public string $recordCompany,
+        #[MapName(new ProvidedNameMapper('country field'))]
+        public string $country,
+        #[MapName(StudlyCaseMapper::class)]
+        public string $cityName,
+        #[MapName(LowerCaseMapper::class)]
+        public string $addressLine1,
+        #[MapName(UpperCaseMapper::class)]
+        public string $addressLine2,
+    ) {
+    }
+}
+```
+```php
+[
+    'name' => 'Rick Astley',
+    'record_company' => 'RCA Records',
+    'country field' => 'Belgium',
+    'CityName' => 'Antwerp',
+    'addressline1' => 'some address line 1',
+    'ADDRESSLINE2' => 'some address line 2',
+]
+```
```

Comment:
Can you move this to a separate page in advanced-usage? And add links to it from this page and https://github.com/spatie/laravel-data/blob/b65bfc03dadb31605bbef4597c2917e2f901f2bc/docs/as-a-data-transfer-object/mapping-property-names.md#L94-L93

Reply:
Yes, I can do it

Reply:
1. I decided to also add a link to the documentation page https://github.com/spatie/laravel-data/blob/main/docs/as-a-resource/mapping-property-names.md as it directly applies to it
2. I hope I have correctly specified the page weight for the menu sorting.